build.ps1

```diff
@@ -0,0 +1,466 @@
+# Default to Debug
+$Configuration = 'Debug'
+
+# Color
+$Success = 'Green'
+$Warning = 'Yellow'
+$Err = 'Red'
+
+Function Error ($msg)
+{
+    Write-Host "[Error:]" $msg -ForegroundColor $Err
+}
+
+Function Warning ($msg)
+{
+    Write-Host "[Warning:]" $msg -ForegroundColor $Warning
+}
+
+Function Success ($msg)
+{
+    Write-Host "[Success:]" $msg -ForegroundColor $Success
+}
+
+Function Info ($msg)
+{
+    Write-Host "[Info:]" $msg
+}
+
+if ($args.Count -eq 0)
+{
+    $TestType = 'Nightly'
+    $Configuration = 'Release'
+}
+elseif ($args[0] -match 'quick' -or ($args[0] -match '-q')) 
+{
+    $TestType = "Quick"
+}
+elseif ($args[0] -match 'DisableSkipStrongName')
+{
+    $TestType = "DisableSkipStrongName"
+}
+elseif ($args[0] -match 'EnableSkipStrongName')
```

Comment:
There is a downside to this version of 'EnableSkipStrongName' and the VSTS build: running this script with 'EnableSkipStrongName' causes all projects to be rebuilt. ODL gets around this by running 'EnableSkipStrongName' as a machine prep step but WebApi has it as part of the build.

The VSTS build tasks should be changed to stop calling this and instead put it in a machine prep build definition.

Reply:
Machine prep definition created, test task modified.

Reply:
Machine prep definition created, test task modified. Machine prep now setting up skip string name verification for all binaries.